### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bradh**
#### What does this PR do?
Documentation typo fix.

#### Who is reviewing it?
@kcwire 
@dcruver 

#### How should this be tested?
Inspection only.

#### Any background context you want to provide?
N/A.

#### What are the relevant tickets?
If you really need one, I can do that.

#### Screenshots (if appropriate)
N/A.

#### Checklist:
- [ Y] Documentation Updated
- [N/A ] Update / Add Unit Tests
- [ N/A] Update / Add Integration Tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Connexta Alliance is an open source, modular integration framework building on D
 ## Alliance Features
  * STANAG 4559 Interoperabilty
     - Federated Source and Endpoint implementations of STANAG 4559 allow interoperability with other systems implementing the spec
- * NITF/NSIF (STANG 4554) Ingest and Parsing
+ * NITF/NSIF (STANG 4545) Ingest and Parsing
```
Comment (jlcsmith) [inline README.md]: 2 typos here - "STANG" should be STANAG

**Comment by kcwire** [comment]
thanks @bradh!

**Comment by bradh** [comment]
Fixed additional typo as pointed out by @jlcsmith

**Comment by bdeining** [comment]
:+1:

**Pull request merged**